picker/src/main/java/me/angrybyte/numberpicker/view/ActualNumberPicker.java

```diff
@@ -287,6 +287,32 @@ public int getMinValue() {
         return mMinValue;
     }
 
+    /**
+     * Sets the minimum value to this number picker.
+     *
+     * @param minValue Minimum value to display by the number picker.
+     */
+    public void setMinValue(int minValue) {
+        if (mMaxValue <= minValue) {
+            throw new RuntimeException("Cannot use min_value " + minValue + " because the max_value is " + mMaxValue);
+        }
+        this.mMinValue = minValue;
+        mValue = (mMinValue > mValue) ? (mMaxValue + mMinValue) / 2 : mValue;
+    }
+
+    /**
+     * Sets the maximum value to this number picker.
+     *
+     * @param maxValue Maximum value to display by the number picker.
+     */
+    public void setMaxValue(int maxValue) {
+        if (maxValue <= mMinValue) {
+            throw new RuntimeException("Cannot use max_value " + maxValue + " because the min_value is " + mMinValue);
+        }
+        this.mMaxValue = maxValue;
+        mValue = (mMaxValue < mValue) ? (mMaxValue + mMinValue) / 2 : mValue;
```

Comment:
I believe you are setting the values upfront to listeners registration. But it is not restricting user to affect the mValue before they are set, so this might be needed.
mmm
I will check this week.